selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -67,16 +68,37 @@ public String getName() {
     return repoName;
   }
 
+  public String getSha1(String branchName) throws IOException {
+    return ghRepo.getBranch(branchName).getSHA1();
+  }
+
   /**
-   * Creates reference to branch, tag, ... from master branch.
+   * Creates reference to the new branch with {@code branchName} from default branch.
    *
-   * @param refName is a name of branch, tag, etc
+   * @param branchName is a name of new branch
```

Comment:
@param branchName name of the branch which should be created

Reply:
OK